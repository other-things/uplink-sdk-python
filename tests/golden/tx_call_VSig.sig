r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 54102028617442523811878602978381777324460573738821930884027884163442227791007
