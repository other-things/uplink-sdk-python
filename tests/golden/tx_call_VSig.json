{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": [
                            "115136800820456833737994126771386015026287095034625623644186278108926690779567", 
                            "98245280522003505644797670843107276132602050133082625768706491602875725788467"
                        ], 
                        "tag": "VSig"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE01NDEwMjAyODYxNzQ0MjUyMzgxMTg3ODYwMjk3ODM4MTc3NzMyNDQ2MDU3MzczODgyMTkzMDg4NDAyNzg4NDE2MzQ0MjIyNzc5MTAwNw=="
}